CMakeLists.txt

```diff
@@ -139,33 +138,40 @@ cppyy_make_python_package(
 
 if("${BUILD_TESTING}")
     set(cxx_tests_dir "${CMAKE_CURRENT_LIST_DIR}/tests/cxx")
-    set(python_tests_dir "${CMAKE_CURRENT_LIST_DIR}/tests/python")
+    set(python_test_dir "${CMAKE_CURRENT_LIST_DIR}/tests/python")
```

Comment:
I'm fine changing this from `tests` to `test`, but if you do that for the Python variable you should also do that for the C++ one.

Reply:
I renamed `cxx_tests_dir` with 3f81504532bf89beffdcae2b1fb91857585cacd0. I did it to be consistent with other repos. But, still there is inconsistency about capitalization of such variables. Do we have a rule?

Reply:
I don't think we had a convention specifically referring to the `test`/`tests` directory (although it would be good to be consistent). With respect to capitalization I think we primarily were using using snake_case for CMake variables not defined by CMake itself.